src/docfx/Errors.cs

```diff
@@ -685,6 +685,18 @@ public static Error BookmarkNotFound(SourceInfo? source, Document reference, str
             /// </summary>
             public static Error Custom404Page(Document file)
                 => new Error(ErrorLevel.Warning, "custom-404-page", $"Custom 404 page will be deprecated in future. Please remove the 404.md file to resolve this warning", file.FilePath);
+
+            /// <summary>
+            /// Html Tag value must be in allowed list
+            /// </summary>
+            public static Error UnSafeHTMLTag(Document file, string tag)
```

Comment:
nit: keep method name same as error code

```suggestion
            public static Error DisallowedHtml(Document file, string tag)
```

Reply:
got it